### ISSUE TITLE
Book rental form

### ISSUE THREAD

**Issue by TealTurtle** [dsantosp12/Textrade#69]
"Are there markings inside the book? (Notes in margins etc.?)"

-Maybe provide the option to say how much
-Maybe put the word "yes" next to the check box

**Comment by dsantosp12** [comment]
Yeah I can work with that, I encountered that problem when merging the front-end I believe that the files that @ninapetrop pushed are fine, it was just me breaking it :+1:  lol

**Comment by TealTurtle** [comment]
ah ok :)
